.gitignore

```diff
@@ -31,9 +31,9 @@ profile.iml
 profile.ipr
 profile.iws
 /.datastore
-.idea/
```

Comment:
nitpick why removing .idea from .gitignore https://www.toptal.com/developers/gitignore/api/go,intellij seems legit

Reply:
Intellij warned me of it being a duplicate entry along with `.history/`. We have an entries for `.idea/` and `.history/` in ln:7 and ln:20 respectively.

Reply:
awesome thanks @egaxhaj-figure